app/models/concerns/account_settings.rb

```diff
@@ -12,26 +15,33 @@ module AccountSettings
     end
 
     setting :allow_signup, type: 'boolean', default: true
-    setting :bulkrax_validations, type: 'boolean'
+    setting :bulkrax_validations, type: 'boolean', disabled: true
```

Comment:
disabled here doesn't _do_ anything yet. it just identifies those setting that Hyku doesn't use at the moment. Most of these are things that are in hyku_addons or other places but haven't made it in to hyku yet.  I'm designating them so I can skip them in the UI until they are doing something.